Решение второго пункта курсового проекта

segakarel opened the pull request.

segakarel committed:
Решение второго пункта курсового проекта

ShumilinDmA commented:
Нужно вывести
1) Геометрические величины ступени. Аля длинны лопаток, углы, диаметры.
2) Нужно вывести термрдинамические точки и собрать в табличку или какой-то читаемый вид для сравнения
3) Нужно табличку с треугольниками скоростей. Как минимум показать что за углы и как выбрался профиль
4) Нужно показать потери
5) Прочностные показатели

ShumilinDmA commented:
Вероятнее всего где-то ошибка в расчете U/Cф еще. График выглядит странно

segakarel committed:
Delete 2task_v1.1/G_2.py

segakarel committed:
Delete 2task_v1.1/Course_project.ipynb

segakarel committed:
Изменения по дополнению кода

Таблицы потерь, процесса расширения, геометрии, вывод силовых напряжений, добавлена hs-диаграмма расширения

ShumilinDmA commented on 2task_v1.1/G_2.py:
```diff
@@ -0,0 +1,1473 @@
+        
+        
+        # Plot velocity triangles if needed
+        if diameter == 1.1:
```
Эм
А если я поменяю параметры расчета, угол какой-нибудь, ты тоже будешь внутри что-то исправлять?

ShumilinDmA commented:
1) Где выбор теплоперепада под максимум кпд?
2) Где длинны лопаток?

Кажется в целом тот результат что сейчас презентуется не соотносится с ступенью при максимуме кпд. Кстати да, вопрос на понимание. Проектируем ступень под какое максимальное КПД, лопаточное или внутренее? У тебя выводятся таблички самого последнего расчета в цикле, а не максимального. В прицнипе это оч плохо что вы все интегрировали вывод результатов как-то совместно с отрисовкой графиков КПД. В целом идеалогия расчета бредовая.

Адекватно будет выглядеть так:

1) Строим графики КПД. Выбираем параметр Dср. Пишем, что выбрали такое то Dcр
2) На него начинаем расчет ступени. После получения треугольников скоростей забиваем параметры лопаток, так как заранее ты не можешь это узнать не построив их.
3) Считаешь все остальные параметры ступени отдельной и их презентуешь. Шаг за шагом.
